templates/wg.conf.j2

```diff
@@ -68,3 +68,26 @@ SaveConfig = true
     {% endif %}
   {% endif %}
 {% endfor %}
+{% if wireguard_unmanaged_peers is defined %}
+
+    # Peers not managed by ansible from wireguard_unmanaged_peers
+    {% for peer in wireguard_unmanaged_peers.keys() %}
+    [Peer]
+    # {{ peer}}
+    {% if wireguard_unmanaged_peers[peer].public_key is defined %}
+    PublicKey = {{wireguard_unmanaged_peers[peer].public_key}}
```

Comment:
```suggestion
    PublicKey = {{ wireguard_unmanaged_peers[peer].public_key|trim }}
```
This should be changed for L81,84,87 and 90 too.

Reply:
I don't see trim on other fields so why specifically for wireguard_unmanaged_peers?

Reply:
The main idea here was that `trim` is a good idea in general but esp. if the variable was provided directly by the user and is not the result of some computation like the private/public keys that are automatically created e.g. I want to add this at other places too in the future where it makes sense.

Reply:
I fixed the whitespace but didn't add trim - if it's added I think it makes sense to add on other variables too.